packages/expo-dev-menu/ios/OverlayContainer/Internal/HeightContraintOverlayTranslationController.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  HeightConstraintOverlayTranslationController.swift
+//  OverlayContainer
+//
+//  Created by Gaétan Zanella on 29/11/2018.
```

Comment:
If we do not own the copyright, we should reproduce the license (MIT, etc.) here